src/main/java/nextstep/subway/path/application/PathService.java

```diff
@@ -0,0 +1,50 @@
+package nextstep.subway.path.application;
+
+import nextstep.subway.line.domain.LineRepository;
+import nextstep.subway.path.dto.PathResponse;
+import nextstep.subway.path.exception.StationNotFoundException;
+import nextstep.subway.path.exception.StationsSameException;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.domain.StationRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+public class PathService {
+
+    private LineRepository lineRepository;
+    private StationRepository stationRepository;
+
+    public PathService(LineRepository lineRepository, StationRepository stationRepository) {
+        this.lineRepository = lineRepository;
+        this.stationRepository = stationRepository;
+    }
+
+    @Transactional(readOnly = true)
+    public PathResponse findPath(Long source, Long target) {
+        checkValid(source, target);
+        PathFinder pathFinder = new PathFinder(lineRepository.findAll());
+
+        Station sourceStation = getStation(source);
+        Station targetStation = getStation(target);
+
+        List<Station> pathStations = pathFinder.getPathStations(sourceStation, targetStation);
+        int distance = pathFinder.getDistance(sourceStation, targetStation);
```

Comment:
```suggestion
        List<Station> pathStations = pathFinder.getPathStations(sourceStation, targetStation);
        int distance = pathFinder.getDistance(sourceStation, targetStation);
```
`pathStations`, `distance` 별도로 받는것도 괜찮지만, 
결과 값에 대해 하나의 객체로 받는것에 대해 어떻게 생각하시나요? :thinking: